features/steps/user-management.rb

```diff
@@ -0,0 +1,281 @@
+    change_mutable_fields(@another_user)
+  end
+
+  step 'I change the admin checkbox' do
+    if @user.admin?
+      @old_admin_value = true
+      uncheck "user_admin"
+    else
+      @old_admin_value = false
+      check "user_admin"
+    end
+  end
+
+  step 'I click Update' do
+    click_on "Update"
+    sync_page
+  end
+
+  step 'the mutable fields are not changed' do
+    expect(find(".user_name")).to have_content(@user.name)
+    expect(find(".user_tone_name")).to have_content(@user.tone_name)
+    expect(find(".user_email")).to have_content(@user.email)
+    expect(find(".user_start_date")).to have_content(@start_date)
+  end
+
+  step 'the mutable fields are changed' do
+    expect_mutable_fields_to_be_changed(@user)
+  end
+
+  step 'the mutable fields for that other user are changed' do
+    expect_mutable_fields_to_be_changed(@another_user)
+  end
+
+  step 'the admin field is not changed' do
+    if @old_admin_value
+      expect(page).to have_css("p.user_admin")
+    else
+      expect(page).not_to have_css("p.user_admin")
+    end
+  end
+
+  step 'the admin field is changed' do
+    id_class = ".user-id-#{@another_user.id}"
+    if @old_admin_value
+      expect(page).not_to have_css("p#{id_class}.user_admin")
+    else
+      expect(page).to have_css("p#{id_class}.user_admin")
+    end
+  end
+
+  step 'I click Cancel' do
+    click_on "Cancel"
+    sync_page
+  end
+
+  step 'I click Sign In' do
+    click_on "Sign in"
+    sync_page
+  end
+
+  step 'I am still in the admin account' do
+    find("li.navigation").hover
+    expect(find("li.informational")).to have_content(@user.tone_name)
+  end
+
+  step 'I enter a registered email' do
+    @user = FactoryGirl.create(:user, password: PASSWORD)
+    fill_in("user_email", with: @user.email)
+  end
+
+  step 'I enter the associated password' do
+    fill_in("user_password", with: PASSWORD)
+  end
+
+  step 'I enter an unregistered email' do
+    fill_in("user_email", with: "not_registered@user.com")
+  end
+
+  step 'I enter an invalid password' do
+    fill_in("user_password", with: "incorrect-password")
+  end
+
+  private
+
+  def change(item)
+    item[0] + "changed" + item[1..-1]
+  end
+
+  def select_date(date)
+    find("#user_band_start_date_1i").select(Date.parse(date).year)
+    find("#user_band_start_date_2i").select(Date::MONTHNAMES[Date.parse(date).month])
+    find("#user_band_start_date_3i").select(Date.parse(date).day)
+  end
+
+  def change_mutable_fields(user)
+    @changed_date = "1-Apr-#{Date.today.year + 1}" # app always allows one year more than today
+    fill_in "user_name",       with: change(user.name)
+    fill_in "user_tone_name",  with: change(user.tone_name)
+    fill_in "user_email",      with: change(user.email)
+    select_date(@changed_date)
+  end
+
+  def expect_mutable_fields_to_be_changed(user)
+    id_class = ".user-id-#{user.id}"
+    expect(find("#{id_class} .user_name").text).to       match("#{change(user.name)}")
```

Comment:
I might, before the UI update on the user, call `user.attributes.slice(:name, :tone_name, :email)` and save that to a variable. Here, then, I can check that the UI equals the transformed values of those hash keys...or more correctly, that the UI does not match the untransformed values of those hash keys.